packages/react-html/README.md

```diff
@@ -276,7 +276,9 @@ const markup = render(<Html>Hello world!</Html>);
 
 ### `showPage()`
 
-This function encapsulates the logic for showing the page in development, where it is hidden on the initial render by default. You must call this function from your client entry point, usually right after hydrating your React app. It returns a promise that resolves after the document is guaranteed to be visible.
+This function encapsulates the logic for showing the page in development, where it is hidden on the initial render by default. This avoids flashes of unstyled content that are an unavoidable side effect of embedding CSS in JavaScript.
+
+You must call this function from your client entry point, usually right after hydrating your React app. It returns a promise that resolves after the document is guaranteed to be visible.
```

Comment:
There's a useful code snippet higher up in the doc in the [In your client entrypoint
](https://github.com/Shopify/quilt/blob/master/packages/react-html/README.md#in-your-client-entrypoint) section.  It might be helpful to readers if we link up to that.

Reply:
+1 for showing a short example